### ISSUE TITLE
md-chips is not working on 0.7.1

### ISSUE THREAD

**Issue by daviddelucca** [vuematerial/vue-material#852]
I'm using VueJS 2.2.6 and VueMaterial 0.7.1 because I need pagination working but chips is not working.

The value typed on chips is added to array but is showing an empty string.

On version 0.7.3 `md-chipś` is working but pagination not (same for version 0.7.2).

![printscreen](https://user-images.githubusercontent.com/5162985/27443396-43239296-5749-11e7-9d7a-aedd49a50fd4.png)

```
<md-chips v-model="contacts" md-input-placeholder="Add a contact"></md-chips>

export default {
  name: 'aggregated-raw-events',
  data () {
    return {
      contacts: ['Marcos Moura']
    }
  }
}
```

**Comment by pablohpsilva** [comment]
I'll look into it ASAP, but, AFAIK it is working. Check it out:
![jun-23-2017 08-39-44](https://user-images.githubusercontent.com/2090635/27480751-b71c4200-57ef-11e7-99cc-b13bc9ed4684.gif)

**Comment by pablohpsilva** [comment]
Probably fixed on #856
